recipes/vc/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: vc
+  version: 9   # [win and py27]
+  version: 10  # [win and py34]
+  version: 14  # [win and py35]
+
+build:
+  number: 0
+  track_features:
+    - vc9      # [win and py27]
+    - vc10     # [win and py34]
+    - vc14     # [win and py35]
+  skip: true   # [unix]
+
+requirements:
+  build:
+    - python
+
+  run:
+    - vs2008_runtime  # [win and py27]
+    - vs2010_runtime  # [win and py34]
+    - vs2015_runtime  # [win and py35]
```

Comment:
If we did not have these `[win and py27]` selectors, we could not do this. So I think the best way would be to go the hard way and create 3 meta packages. Then we can also have `vc11`, `vc12` and any other version if needed in future.